share/spice/nobel_prize/nobel_prize.js

```diff
@@ -0,0 +1,81 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_nobel_prize = function(api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || !api_result.prizes || api_result.prizes.length == 0) {
+            return Spice.failed('nobel_prize');
+        }
+
+        function getSearchTerm() {
+            var script = $('[src*="/js/spice/nobel_prize/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/nobel_prize\/([^\/]+)/)[1],
+            match = decodeURIComponent(query).match(/(\d{4})?-(\w+)?/),
+            year = match && match[1],
+            category = match && match[2],
+            searchTerm = ((year && 'Year: ' + year) || "") 
+                        + " " 
+                        + ((category && ' Category: ' + category.toUpperCase()) || "");
+            
+            return [searchTerm, year, category];
+        }
+
+        function getDetailTitleNames(item) {
+            var names = [];
+            for (var laureate of item.laureates) {
```

Comment:
looks like `for...of` isn't supported by IE. Please use `$.each()`.

It may be easier to build an array of names and use `Handlebars.helpers.concat()` (https://github.com/duckduckgo/duckduckgo-template-helpers/blob/master/template_helpers.js#L57) to build the output.

`Handlebars.helpers.concat(array, { sep: ",", conj: "and" })`